terra/assetlist.json

```diff
@@ -6509,7 +6509,55 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.png",
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.svg"
         }
+      ],
+      "base": "cw20:terra1m58hc296srr6xygrrfyu6u0e32pl8d459nfs55pmegke8zlv94rqsjz882",
+      "name": "CatWifHat Project",
+      "display": "CatWifHat",
+      "symbol": "CWH",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/cwh.jpg"
```

Comment:
image format mismatch (must be png for "png" property (svg also accepted)). jpg not accepted.